bundles/org.openhab.core.automation.module.script.rulesupport/src/main/java/org/openhab/core/automation/module/script/rulesupport/internal/loader/ScriptFileReference.java

```diff
@@ -116,4 +113,21 @@ public int compareTo(ScriptFileReference other) {
             return 0;
         }
     }
+
+    @Override
+    @NonNullByDefault({})
```

Comment:
Why did you add the annotation here? I think it is not needed.

Reply:
This is because it overrides the object method where the parameter is nullable. Saying that, it's cleaner to explicitly mark the parameter this way, so I've switch to that instead.